pdns/dnsparser.hh

```diff
@@ -304,6 +305,18 @@ struct DNSRecord
   uint16_t d_clen{};
   DNSResourceRecord::Place d_place;
 
+  [[nodiscard]] std::string toString(const std::string& indent = "") const
```

Comment:
this does not look like toString for a record, more like some special represetantion.

Reply:
You mean it should be renamed to something else?

Reply:
yes.